lib/fastlane/plugin/wpmreleasetoolkit/actions/common/publish_release_action.rb

```diff
@@ -0,0 +1,63 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class PublishReleaseAction < Action
+      def self.run(params)
+        repository = params[:repository]
+        name = params[:name]
+        prerelease = params[:prerelease]
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+
+        url = github_helper.publish_release(
+          repository: repository,
+          name: name,
+          prerelease: prerelease
+        )
+        UI.success("Successfully published GitHub Release #{name}. You can see it at '#{url}'")
+        url
+      end
+
+      def self.description
+        'Publishes an existing GitHub Release still in draft mode'
```

Comment:
💄  I know that we haven't been super consistent across our release-toolkit actions about this, but I'd consider using infinitive form here for the `description`?

```suggestion
        'Publish an existing GitHub Release still in draft mode'
```